CoreGraphFramework/src/au/gov/asd/tac/constellation/graph/attribute/AbstractObjectAttributeDescription.java

```diff
@@ -89,39 +96,48 @@ public void setCapacity(final int capacity) {
             Arrays.fill(data, len, capacity, defaultValue);
         }
     }
-    
+
     @Override
     public String getString(final int id) {
-        return data[id] != null ? String.valueOf((T) data[id]) : null;
+        return String.valueOf((T) data[id]);
     }
-    
+
     @Override
     public void setString(final int id, final String value) {
         data[id] = convertFromString(value);
     }
-    
+
+    @Override
+    public String acceptsString(final String value) {
+        try {
+            convertFromString(value);
+            return null;
+        } catch (final Exception ex) {
```

Comment:
I originally did convert this to specific exceptions, however later thought that it'd be safer to catch all exceptions since `convertFromString` is an overridable method.

Reply:
I still think my preference would be catch the specific exceptions rather than the generic catch-all and then adjust the catches as need be. If for no other reason, sonar has a rule for avoiding catching generic exceptions and so if we can avoid introducing unnecessary code smells then that would be a preference

Reply:
I've set `convertFromString` with a `throws `IllegalArgumentException`, and i'm now explicitely catching that. The problem is that we can't predict what the implementation will actually throw in terms of runtime exceptions, so if anything unplanned happens you will get a nasty popup instead of a message saying the value is invalid. Does this work for you @arcturus2, @antares1470?

Reply:
I'll leave the call to @arcturus2. The nasty pop up isn't ideal but may be a necessary risk. In theory, we could apply this thinking to a lot of functions in the code base but I'm not sure we want to take it that far